pytransferwise/borderless_account.py

```diff
@@ -19,7 +19,7 @@ class BorderlessAccount(object):
     def list(self, profile_id):
         return munchify(self.service.list(params={"profileId": profile_id}))
 
-    def statement(self, profile_id, account_id, currency, interval_start, interval_end):
+    def statement(self, profile_id, account_id, currency, interval_start, interval_end, statement_type='COMPACT'):
```

Comment:
A few small questions related to `statement_type`:

- Did you consider a default value of `None` as an alternative?  It looks like TransferWise might choose `COMPACT` for us as the default 'server-side' if we leave it empty.

- Could / should we validate that `statement_type` is one of: `[COMPACT, FLAT]`?  Or should we rely on the API returning an error in that case?  (what would happen?)